src/main.c

```diff
@@ -411,7 +411,7 @@ void assert_nrf_callback (uint16_t line_num, uint8_t const * p_file_name)
 // Process BLE event from SD
 uint32_t proc_ble(void)
 {
-  __ALIGN(4) uint8_t ev_buf[ BLE_EVT_LEN_MAX(BLEGATT_ATT_MTU_MAX) ];
+  __ALIGN(4) uint8_t ev_buf[ BLE_EVT_LEN_MAX(BLEGATT_ATT_MTU_MAX) ] = {0};
```

Comment:
It is not worth the effort to initialize the whole buffer, there is error check for sd_ble_evt_get() already. We should fine another way to keep gcc happy.

Reply:
How about using pragma, if you are confident it is mistaken?
```c
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
...  // ignore the diagnostic for code here
#pragma GCC diagnostic pop
```

Reply:
@dhalbert It's also good idea, but I don't like using a per-preprocessor directives and makes it compiler specific.
Right now I just initialized header parameter directly. Maybe it's not a cleanest solution, but right now i don't have any other ideas.